### DISCUSSION TITLE
how `indent.get` function work

### DISCUSSION THREAD

**Discussion by shellRaining** [lukas-reineke/indent-blankline.nvim#898] [Q&A] [answered]
hello, I'm learning how `indent.get` working in `ibl`, and I found you write a test case below

https://github.com/lukas-reineke/indent-blankline.nvim/blob/d98f537c3492e87b6dc6c2e3f66ac517528f406f/specs/features/indent_spec.lua#L63-L67

my question is why the table has `TAB_START_SINGLE`, other than `TAB_START TAB_FILL TAB_END`, I think the blank line is display like this

blank blank tab(two char width) blank tab(three char width), Is my understanding wrong?

**Comment by lukas-reineke** [comment, marked as answer]
You are correct, this is a bug.
Tabs need to reset the count of spaces in `indent.get`
I made a fix #899
